code/modules/admin/IsBanned.dm

```diff
@@ -81,13 +81,17 @@
 							addclientmessage(ckey,"<span class='adminnotice'>Admin [key] has been allowed to bypass a matching non-admin ban on [i["key"]] [i["ip"]]-[i["computerid"]].</span>")
 						continue
 				var/expires = "This is a permanent ban."
+				var/global_ban = "This is a global ban."

```

Comment:
This in particular needs to be worded in a way that makes it clear they're banned from all *Bee* servers but not all *SS13* servers or anything else like that.

Otherwise, this PR is good to go.